.github/workflows/tests.yml

```diff
@@ -169,3 +169,202 @@ jobs:
           export -f _run_tests
 
           find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -not -wholename '*/Bridge/PhpUnit/*' -print0 | xargs -0 -n1 dirname | sort | parallel _run_tests
+
+  high_deps:
+    name: PHPUnit (7.4, high deps)
+    runs-on: Ubuntu-20.04
+
+    env:
+      SYMFONY_DEPRECATIONS_HELPER: weak
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          ini-values: "memory_limit=-1"
+          php-version: "7.4"
+          extensions: amqp, apcu, igbinary, intl, mbstring, memcached, mongodb, redis
+          tools: flex
+
+      - name: Configure composer
+        run: |
+          SYMFONY_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*')
```

Comment:
Should we really keep .travis.yml?

Reply:
I don't want to create another place that we have to keep updated. If Travis goes away completely, we'll find another way. For now, this is consistent with other GHA jobs on this repo.